pandas/core/arrays/datetimelike.py

```diff
@@ -468,6 +542,67 @@ def _values_for_factorize(self):
     def _from_factorized(cls, values, original):
         return cls(values, dtype=original.dtype)
 
+    def _values_for_argsort(self):
+        return self._data
+
+    # ------------------------------------------------------------------
+    # Additional array methods
+    #  These are not part of the EA API, but we implement them because
+    #  pandas assumes they're there.
+
+    def searchsorted(self, value, side='left', sorter=None):
```

Comment:
Hold off on changing the signature `v` to `value` until 
https://github.com/pandas-dev/pandas/pull/24350/files/ff8bbc361b5bed4c244ba38a4a49c0e2e79758e5#diff-99633b1041c2ea6283aa3d5bcf79e8cd is resolved.